test/test_gguf_format.py

```diff
@@ -92,8 +92,8 @@ def test_q4_1(self):
 
         from auto_round.eval.evaluation import simple_evaluate_user_model
         result = simple_evaluate_user_model(model, self.tokenizer, batch_size=16, tasks="openbookqa")
-        # 0.252
-        self.assertGreater(result['results']['openbookqa']['acc,none'], 0.25)
+        # 0.23
+        self.assertGreater(result['results']['openbookqa']['acc,none'], 0.22)
```

Comment:
better not use openbookqa in the future, as it is not very robust based on my experiments.